pkg/scheduler/framework/session.go

```diff
@@ -63,6 +64,9 @@ type Session struct {
 	Queues         map[api.QueueID]*api.QueueInfo
 	NamespaceInfo  map[api.NamespaceName]*api.NamespaceInfo
 
+	NodeMap   map[string]*k8sframework.NodeInfo
```

Comment:
There is a big redundent between Nodes and NodeMap and a little bit confusing for developers.

Reply:
Yes, they work almost like the same, the only difference is the later one is used when calling k8s apis, I just put it here for performance sake. I add a little comments above, hopefully it'll be more clear.